lib/stdlib/src/otp_internal.erl

```diff
@@ -64,6 +64,8 @@ obsolete(net, sleep, 1) ->
     {deprecated, "use 'receive after T -> ok end' instead"};
 obsolete(queue, lait, 1) ->
     {deprecated, "use queue:liat/1 instead"};
+obsolete(ssl, prf, 5) ->
+    {deprecated, "Use export_key_materials/4 instead. Note that in OTP 28 the 'testing' way of calling this function will no longer be supported."};
```

Comment:
maybe wrap line. you reached 147 characters here.

Reply:
This is automatically generated by the script.